single-opt.html

```diff
@@ -13,30 +13,33 @@
     <link href="https://fonts.googleapis.com/css2?family=Limelight&display=swap" rel="stylesheet">
 </head>
 
-<body class="hero is-success is-fullheight">
-    <section id="single-whole">
-        <!-- Hero head: will stick at the top -->
-        <!-- <div class="hero-head"> -->
-        <header class="navbar" id="nav-head">
-            <div class="navbar-end is-flex is-justify-content-end">
+<body class="hero is-fullheight film-banner">
```

Comment:
just check the class naming here. The "film-banner" class name should be in the <nav> element so it only effects the navbar, whereas you have it in the entire body and could cause problems down the line.